src/Classes/ServiceAPI/ServiceAPI.php

```diff
@@ -99,43 +99,12 @@ protected function addMixins(&$data, $mixins, $mixin_funcs, $strict = true)
         }
     }
 
-    public function toDataSource($full = false)
+    // TODO: Convert to abstract function once all child classes use mixins properly
+    public function toDataSource($mixins = [])
     {
         throw new MyRadioException(get_called_class().' has not had a DataSource Conversion Method Defined!', 500);
     }
 
-    /**
-     * Iteratively calls the toDataSource method on all of the objects in the given array, returning the results as
-     * a new array.
-     *
-     * @param array $array
-     * @param bool  $full  If true, will return expanded data if available.
-     *
-     * @return array
-     *
-     * @throws MyRadioException Throws an Exception if a provided object is not a DataSource
-     */
-    public static function setToDataSource($array, $full = false)
```

Comment:
This is still used, in, e.g. Controllers\Charts\default/listChartReleases, Controllers\Podcast\default, Controllers\Quotes\default...